pressbooks/pressbooks-multi-institution
# feat: refine institutions behaviour

Issue #31 

This PR aims to refine the Institution Form behaviour as well as some minor updates.

1. The Institutional Manager combo box will only show users that are not super admins, network managers or institutional managers from other institutions
2. If a regular user is assigned to a different institution and is selected in the combo box, they will get removed from the previous institution and included to the new one
3. It will add validation to the email domains

This PR also updates the indentation of some blade files that were inconsistent with others, and updates the error message handling/rendering.

In addition, the PR updates how we grant/revoke super admin access to institutional managers to happen inside the  `syncManagers` method.

**How to test**

- Follow the steps and check that the features described here and in #31 work as expected
- Make sure that associating an Institutional Manager correctly grants them Super Admin access
- Make sure that removing an Institutional Manager correctly revokes the Super Admin access 
- Test the validation messages for invalid/duplicate domains
